src/app/listing/listing.page.html

```diff
@@ -24,6 +24,16 @@ <h1>{{stateData?.name}}</h1>
         </ion-col>
       </ion-row>
   </ion-list>
+  
+  <ion-segment class="custom-segment" [(ngModel)]="filter" (ionChange)="filterChanged($event)">
+    <ion-segment-button value="assignedToMe">
```

Comment:
I mentioned that these options should come from a constant file right, where is that change made?